### ISSUE TITLE
MOBILE All Components - 1.4.4 Resize Text - Resizing the page to 200% causes all components to overlap one another

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9963]
As a user with vision difficulties, I may need to use the zoom function on mobile to access content. I expect the page components to resize with the page and, at 200%, expect all components to be visible and accessible, with no overlapping content, no content cut off and all content available by scrolling up and down. 

**Linked work item:** 
#9603 

**Expected Result:**
When the user resizes the page to 200%, components should still clearly be visible and accessible, and the user should be able to access all content in the page without having to scroll in two dimensions.

**Actual Result:**
Resizing to 200% currently on mobile causes all components to overlap one another, reducing readability of text and causing some components to disappear behind others. The user is also forced to scroll in two dimensions to reach the left nav.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/99fc43b0-3b24-4900-96b0-c60697526a41)

**Comment by imogenroberts** [comment]
RETEST: The Search results and Pagination components are now reflowing well with nothing obscured, but the map filter icons are cut off at either side of the page at 200%. In addition, there is a lot of white space on the right side of the page when the user zooms in and it is unclear why this is needed as it just allows less room for text and components to reflow easily.

**Evidence:**
![MOBILE All Components - 1 4 4 Resize Text](https://github.com/archesproject/arches/assets/77161085/5d05cff7-6c36-4786-a7cd-a188730f4b4f)

**Comment by imogenroberts** [comment]
@johnatawnclementawn while testing the Map Filter I noticed that a lot of the components on mobile are still overlapping one another quite dramatically at zoom levels over 250% (see below video):



![Image](https://github.com/archesproject/arches/assets/77161085/37c34a1d-7759-49a4-a1ba-274609f7dd73)
